src/Tribe/Main.php

```diff
@@ -3598,47 +2911,6 @@ public function save_organizer_data( $postID = null, $post = null ) {
 			Tribe__Events__API::updateOrganizer( $postID, $data );
 		}
 
-		/**
-		 * Add a new Organizer
-		 *
-		 * @param      $data
-		 * @param null $post
-		 *
-		 * @return int|WP_Error
-		 */
-		public function add_new_organizer( $data, $post = null ) {
```

Comment:
Brought this back and deprecated it. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3043))